### PR TITLE
[FIX] travis: Fix file .travis.yml to use version and virtualenv

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +22,7 @@ install:
   #- pushd /tmp/aeroolib/aeroolib; sudo python setup.py install; popd
 
 script:
-    - travis_run_tests
+    - travis_run_tests ${VERSION}
```
Comment (eLBati) [inline .travis.yml]: According to
https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml
`${VERSION}` is unnecessary

**Comment by moylop260** [comment]
Done!

**Comment by pedrobaeza** [comment]
:+1:

**Comment by eLBati** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**